tests/comfy_cli/command/generate/test_app_lifecycle.py

```
@@ -0,0 +1,332 @@
+def api_key(monkeypatch):
+    monkeypatch.setenv("COMFY_API_KEY", "comfyui-test")
+    return "comfyui-test"
+
+
+@pytest.fixture
+def captured_events(monkeypatch):
+    """Drop-in replacement for the autouse track_event mock that records every
+    call so tests can make assertions on the emitted lifecycle."""
+    events: list[tuple[str, dict]] = []
+
+    def _record(event_name, properties=None, *, mixpanel_name=None):
```

Pylint (4.0.5)

[warning] Unused argument 'mixpanel_name'

(W0613)